Cargo.toml

```diff
@@ -19,7 +16,7 @@ git = "https://github.com/gtk-rs/gdk"
 git = "https://github.com/gtk-rs/gtk"
 
 [features]
-gtk_3_10 = []
+gtk_3_10 = ["gtk/3.10"]
 gtk_3_14 = ["gtk_3_10"]
```

Comment:
Ideally `gtk_3_14` should not be there, I put it in when branching appveyor into 3.6 and 3.14 to try avoiding some confusion. Now it's all confusing anyway so might as well remove 3_14 but it has to start with CI scripts changes.